### ISSUE TITLE
Reverb initial loudness and duration are not physically accurate.

### ISSUE THREAD

**Issue by YurgenSchn** [vlad2305m/Sound-Physics-Fabric#40]
Don't know how to describe it, but well: sound energy should be the same no matter the envinroment size. This means reverb should always start loud.

What i get with the current version of this mod, in small caves, is almost no reverb: stones have high reflectivity, so i should be able to hear it's effect on sounds, with reverbs that last a fraction of a second (giving it a very claustrophobic feel).

Currently, reverbs in small caves do sound short, but too quiet - as if it's just a cutout of the remains of a long reverb. Maybe there needs to be a dynamic reverb falloff curve, based on the environment size and material reflectivity.

Basically: reverbs should **always start as loud**, leaving only **tail duration to be dynamic**. It, then, should take into consideration environment size AND material reflectivity: if bounces are farther = long lasting reverb, if material doesn't bounce as much = less energy per iteration, thus shorter tails).

**Comment by vlad2305m** [comment]
Yeah, I don't like it as well because it only takes number of rays reached the player into account. But I have no experience with audio tuning, so I am doing what I am good at (optimisation) first.

**Comment by thedocruby** [comment]
this can likely be fixed by tuning the effect filters and possibly also tweaking the environment mapping. Should be a simple fix, will improve when i get around to it.
